src/plugins/expressions/public/types/index.ts

```diff
@@ -91,6 +91,7 @@ export interface IExpressionLoaderParams {
   customFunctions?: [];
   customRenderers?: [];
   extraHandlers?: Record<string, any>;
+  inspectorAdapters?: Adapters;
```

Comment:
// brainpick
Isn't having `inspectorAdapters` on `IExpressionLoaderParams` too specific?
Could we use `customFunctions` or any of the other args for this?

Reply:
`customFunctions` are actual functions for the interpreter.

i don't know of a better place to put it for now.